### PR TITLE
Minor visual fixes for #40321

### PR DESCRIPTION
**Pull request by alshakero**
5 mins review.

#### Changes proposed in this Pull Request

1. Remove the dot after the vertical name
2. Only show the vertical-field arrow for 3+ characters
3. Keep the layout stable when the arrow appears or disappears

#### Testing instructions

- **Case 1**: Fill the vertical name and site title, the dot shouldn't appear.
- **Case 2**: Fill the vertical name, verify that the arrow only appears 3 characters in.
- **Case 3**: When the vertical name is filled the transition from 2 to 3 characters (arrow appearing) shouldn't change the layout at all.

### CODE REVIEW

**Comment by sirreal** [comment]
Now that we're left aligned, I don't think jumping on show/hide of SVG arrow is an issue anymore. Shall we just remove the `.`?

https://wpcalypso.wordpress.com/gutenboarding/

**Comment by alshakero** [comment]
> Now that we're left aligned, I don't think jumping on show/hide of SVG arrow is an issue anymore. Shall we just remove the . ?

I think it's still worth it to use visibility instead of adding/removing the whole element to save reflows.

File: client/landing/gutenboarding/onboarding-block/acquire-intent/arrow.tsx [hunk 1]
```diff
@@ -3,8 +3,15 @@
...
+	<svg
+		{ ...props }
+		width="26"
+		height="18"
+		viewBox="0 0 26 18"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
 		<path d="M16 1L24 9M24 9L16 17M24 9H0" stroke="currentColor" strokeWidth="2" />
 	</svg>
```
Comment (sirreal) [inline client/landing/gutenboarding/onboarding-block/acquire-intent/arrow.tsx]: This should use `@wordpress/components` `SVG` and `Path`. Ok to leave that out of scope fo this PR.

Comment (alshakero) [reply client/landing/gutenboarding/onboarding-block/acquire-intent/arrow.tsx]: Thanks! TIL.

**Pull request merged**